tc/core/polyhedral/schedule_tree.h

```diff
@@ -453,6 +453,10 @@ struct ScheduleTree {
     return type_;
   }
 
+  virtual std::ostream& write(std::ostream& os) const {
```

Comment:
Can you make it abstract?  (not at this point, but later in the series)
I'd also pick a name that clarifies that this only prints the contents of a node and
node the entire (sub)tree.  At least at a comment to that effect.

Reply:
Done